tests/plugins/tabletools/cellproperties.js

```diff
@@ -90,7 +76,7 @@
 		},
 
 		// #11439
-		'test load and update field values (#7)': function() {
+		'test load and update field values (#6)': function() {
 			this.doTest( 'table-8', function( dialog ) {
```

Comment:
is `table-8` just the name of the test or does that correspond to sample table or something? If not I guess we can update the number for it as well

Reply:
I believe it's kinda of test fixture in `cellproperties.html` file. I don't know why numbering doesn't match tests one.

Reply:
I see now. looks fine